adapters/dmx/dmx.go

```diff
@@ -271,10 +277,15 @@ func fetchParams(params dmxExt, inst openrtb.Imp, ins openrtb.Imp, imps []openrt
 		tempimp.Secure = &intVal
 	}
 	if banner != nil {
+		if banner.H == nil || banner.W == nil {
+			banner.H = &banner.Format[0].H
+			banner.W = &banner.Format[0].W
+		}
```

Comment:
Please add json test coverage for the `if banner.H == nil || banner.W == nil` condition.